components/JobSummary.js

```diff
@@ -1,45 +1,99 @@
 import ApplyJobButton from "./buttons/ApplyJob";
+import moment from "moment";
 
-export default function JobSummary({ job, className = '' }) {
+export default function JobSummary({ job, className = '', extend }) {
   return (
-    <div className={`bg-white shadow overflow-hidden sm:rounded-lg ${className}`}>
+    <div
+      className={`bg-white shadow overflow-hidden sm:rounded-lg ${className}`}
+    >
       <div className="px-4 py-5 sm:px-6">
         <dl>
           <dt className="sr-only">Location</dt>
           <dd className="flex mt-1 text-sm text-gray-900">
-            <svg viewBox="0 0 20 20" fill="currentColor" className="h-5 w-5 text-gray-400"><path fillRule="evenodd" d="M5.05 4.05a7 7 0 119.9 9.9L10 18.9l-4.95-4.95a7 7 0 010-9.9zM10 11a2 2 0 100-4 2 2 0 000 4z" clipRule="evenodd"></path></svg>
+            <svg
+              viewBox="0 0 20 20"
+              fill="currentColor"
+              className="h-5 w-5 text-gray-400"
+            >
+              <path
+                fillRule="evenodd"
+                d="M5.05 4.05a7 7 0 119.9 9.9L10 18.9l-4.95-4.95a7 7 0 010-9.9zM10 11a2 2 0 100-4 2 2 0 000 4z"
+                clipRule="evenodd"
+              ></path>
+            </svg>
             <div className="ml-4">
-              {!job.location || job.location === 'remote' && <p>Remote</p>}
-              <p className="text-gray-700">{!job.region || job.region === 'any' ? 'Anywhere (on planet Earth)' : job.region}</p>
+              {!job.location || (job.location === 'remote' && <p>Remote</p>)}
+              <p className="text-gray-700">
+                {!job.region || job.region === 'any'
+                  ? 'Anywhere (on planet Earth)'
+                  : job.region}
+              </p>
             </div>
           </dd>
         </dl>
       </div>
+      {!extend && (
+        <div className="border-t border-gray-200 px-4 py-5 sm:px-6">
+          <dl>
+            <dt className="sr-only">Employment type</dt>
+            <dd className="flex mt-1 text-sm text-gray-900">
+              <svg
```

Comment:
This review is kinda not related to the PR :( You can open an issue for that

Reply:
But, this issue is not in the official documentation of the AsyncAPI in the website. Maybe there's just some small error you missed while making the Docs Nav, and this can be fixed using the reference of Docs Nav in documentation.

Reply:
Docs Nav? I'm typically reusing the existing one